vars/capsulePipelineWrapper.groovy

```diff
@@ -1,45 +1,45 @@
 def call() {
     writeConfigs = { envName ->
-        sh label: "generate templates: ${envName}", script: '''
+        sh label: "generate templates: ${envName}", script: """
             vegacapsule template genesis \
-                --home-path ${CONFIG_HOME} \
-                --path ${TEMPLATES_HOME}/genesis.tmpl.json ${FLAGS}
+                --home-path '""" + CONFIG_HOME + """' \
```

Comment:
Can you tell me what's the advantage of this escaping over one of the below:
1. Simply using `’`
2. `${env.CONFIG_HOME}`